packages/editor-ui/src/components/Node.vue

```diff
@@ -169,9 +222,38 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 		return {
 			isTouchActive: false,
 			nodeSubtitle: '',
+			showWebhookNodeTooltip: false,
 		};
 	},
 	methods: {
+		isWebhookNode (): INodeTypeDescription | boolean {
+			const node = this.$store.getters.allNodeTypes.filter((elem: INodeTypeDescription) => {
+				if (elem.displayName === this.data.name) {
+					return elem.group.includes('trigger');
+				} else {
+					return;
+				}
+			});
+
+			return node.length > 0 ? true : false;
+		},
+		isSingleWebhookNode (): INodeUi[] | INodeTypeDescription | boolean {
+			const nodeTypes = this.$store.getters.allNodeTypes;
+			const nodes = this.$store.getters.allNodes;
+			const webhookNodes = [];
+
+			nodeTypes.forEach((types : INodeTypeDescription) => {
+				nodes.forEach((node: INodeUi) => {
+					if (types.name === node.type) {
+						if (types.group.includes('trigger')) {
+							webhookNodes.push(node);
+						}
+					}
+				});
+			});
+
+			return webhookNodes.length === 1;
+		},
```

Comment:
this should be moved to the store.. maybe `getActiveWorkflowTriggerNodes` getter that returns a list of active workflow triggers... you can also use the `nodeType` getter to get the node type instead of looping over all node types.

Reply:
The fix for the scenario with one disabled and one not disabled node (from the same type) is included in the last push. Also I created  getActiveWorkflowTriggerNodes getter in vuex that return a list of active workflow triggers. The unnecessary loop is removed and instead of it, I use nodeType as you suggest.